### PR TITLE
ci: fix "deno fmt --check"

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: core/ops_builtin.rs [hunk 1]
```diff
@@ -287,6 +287,10 @@ async fn op_read_all(
   Ok(vec)
 }
 
+//
+//
+//
+
```
Comment (mmastrac) [inline core/ops_builtin.rs:293]: What is this for?

**Review by mmastrac — APPROVED** [review]
LGTM

File: core/ops_builtin.rs (hunk 1 above)
Comment (bartlomieju) [reply core/ops_builtin.rs:293]: I was debugging something else and it's a leftover.

**Comment by bartlomieju** [comment]
mac 13m -> 10m
linux 13m -> 11m
Windows 13m -> 12m

**Pull request merged**